frontend/src/composables/services/course.service.ts

```diff
@@ -60,21 +62,30 @@ export function useCourses(): CoursesState {
     }
 
     async function createCourse(courseData: Course): Promise<void> {
+        const { t } = i18n.global;
+        const { addSuccessMessage } = useMessagesStore();
+
         const endpoint = endpoints.courses.index;
-        await create<Course>(
-            endpoint,
-            {
-                id: courseData.id,
-                name: courseData.name,
-                description: courseData.description,
-                excerpt: courseData.excerpt,
-                academic_startyear: courseData.academic_startyear,
-                private_course: courseData.private_course,
-                faculty: courseData.faculty?.id,
-            },
-            course,
-            Course.fromJSON,
-        );
+        try {
+            await create<Course>(
+                endpoint,
+                {
+                    id: courseData.id,
+                    name: courseData.name,
+                    description: courseData.description,
+                    excerpt: courseData.excerpt,
+                    academic_startyear: courseData.academic_startyear,
+                    private_course: courseData.private_course,
+                    faculty: courseData.faculty?.id,
+                },
+                course,
+                Course.fromJSON,
+            );
+            addSuccessMessage(
+                t('toasts.messages.success'),
+                t('toasts.messages.courses.create.success', [course.value?.name]),
+            );
+        } catch {}
```

Comment:
Could it also be possible to fill up de catch with a message?
Or is this done with an error in the "create" function?